src/tls/test-certfile.c

```diff
@@ -127,6 +127,7 @@ static void
 test_certfile_multithreaded (void)
 {
   GError *error = NULL;
+  GThread *threads[50];
   int connections[50];
```

Comment:
Since it's now used for two arrays, the `50` here could be given a name, and used for the loops below.

Reply:
This is true.  I was wondering if you'd say something about it :)

Reply:
I combined the two variables into a struct instantiated as one common array :)